app/views/characters/search.haml

```diff
@@ -21,9 +21,15 @@
             %tr
               %td= label_tag :template_id, 'Template'
               %td= select_tag :template_id, options_from_collection_for_select(@templates, :id, :name, params[:template_id]), {class: 'chosen-select', prompt: '— Choose Template —'}
+            %tr
+              %td= label_tag :setting_id, 'Setting'
+              %td= select_tag :setting_id, options_from_collection_for_select(@settings, :id, :name, params[:setting_id]), {class: 'chosen-select', prompt: '— Choose Setting —'}
             %tr
               %td= label_tag :name, 'Name'
-              %td= text_field_tag :name, params[:name], style: 'width: 100%; margin: 5px 0px; box-sizing: border-box;'
+              %td= text_field_tag :name, params[:name], style: 'width: 100%; margin: 5px 0px;'
+            %tr
+              %td= label_tag :pb, 'Facecast'
```

Comment:
This isn't implemented yet right?

Reply:
...I assumed it would inherit from us telling literally all our css to use border-box sizing unless otherwise specified?

Reply:
No sorry I mean I don't see facecast code in the controller

Reply:
https://github.com/Marri/glowfic/pull/1192/files#diff-0bc76a36eb563abbcadfb9f6fd2fdb85R322
which is, granted, _incorrect_, but

Reply:
Oh! In that case this is fine but linked code needs %s